application/src/main/java/org/togetherjava/tjbot/commands/CommandHandler.java

```diff
@@ -50,16 +58,36 @@ public void onReady(@NotNull ReadyEvent event) {
                             new CommandData(command.getCommandName(), command.getDescription())))
                         .queue();
                 }
+            }, throwable -> {
+                handleErrors(throwable, guild);
             });
         });
     }
 
+    private static void handleErrors(Throwable throwable, Guild guild) {
+        new ErrorHandler().handle(ErrorResponse.MISSING_ACCESS, errorResponseException -> {
+            // * Hard to read, there's no other accurate way to get a channel you've the write
+            // * permission in.
+            guild.getTextChannelCache().stream().filter(textChannel -> {
+                return guild.getPublicRole().hasPermission(textChannel, Permission.MESSAGE_WRITE);
+            }).findFirst().ifPresent(textChannel -> {
+                textChannel
+                    .sendMessage("This bot needs the commands scope, please invite it correctly!\n"
+                            + "You can join https://discord.gg/UJBwS2Hvcx for more info, I'll leave now.")
```

Comment:
What is this url? May be a constant

Reply:
Together Java Discord, should be a constant in the main/application/whatever the main is called in that case imo

Reply:
Should also be moved out into the "config system" #100

Reply:
See PR #142 which will add a `discordGuildInvite` property for this.